apps/dotcom/client/src/pages/noindex.tsx

```diff
@@ -0,0 +1,13 @@
+import { Helmet } from 'react-helmet-async'
+import { Outlet } from 'react-router-dom'
+
+export function Component() {
+	return (
+		<>
+			<Helmet>
+				<meta name="robots" content="noindex" />
```

Comment:
looking into this a bit, i think we can do to make this stronger:
`<meta name="robots" content="NOINDEX, NOIMAGEINDEX, NOFOLLOW" />`